dht/i_partitioner.cc

```diff
@@ -48,7 +48,8 @@ sharder::token_for_next_shard(const token& t, shard_id shard, unsigned spans) co
 }
 
 std::ostream& operator<<(std::ostream& out, const decorated_key& dk) {
-    return out << "{key: " << dk._key << ", token:" << dk._token << "}";
+    fmt::print(out, "{{key: {}, token: {}}}", dk._key, dk._token);
```

Comment:
please note, this still breaks the dtest, where we don't have a space after "token:", see https://github.com/scylladb/scylla-dtest/blob/760728eec06fae49954811146f5c1e1af54c63d4/tools/cdc_utils.py#L110

Reply:
i will create yet another fix, if this one manages to land on master.

Reply:
In this case, the dtest shouldn't be that picky about the message format, in particular not about whitespace.
I'd extend the expression in the test to be insensitive to that.

Reply:
makes sense to me. lemme create a fix on dtest side.

Reply:
created https://github.com/scylladb/scylla-dtest/pull/3139